prow/plugins/label/label.go

```diff
@@ -31,8 +31,10 @@ import (
 const pluginName = "label"
 
 var (
-	labelRegex              = regexp.MustCompile(`(?m)^/(area|committee|kind|priority|sig|triage|label)\s*(.*)$`)
-	removeLabelRegex        = regexp.MustCompile(`(?m)^/remove-(area|committee|kind|priority|sig|triage|label)\s*(.*)$`)
+	labelRegex              = regexp.MustCompile(`(?m)^/(area|committee|kind|priority|sig|triage)\s*(.*)$`)
+	removeLabelRegex        = regexp.MustCompile(`(?m)^/remove-(area|committee|kind|priority|sig|triage)\s*(.*)$`)
+	customLabelRegex        = regexp.MustCompile(`(?m)^/label\s*(.*)$`)
```

Comment:
wouldn't this match `/label approved` as well?

Reply:
tried in https://github.com/kubernetes/community/issues/1959#issuecomment-395474359 though, didn't work :)

Reply:
/cc @dims @cblecker

Reply:
just for the record: this was added in https://github.com/kubernetes/test-infra/pull/8015, not this PR

Reply:
`/label approved` doesn't work because `/label foo` only applies the `foo` label if `foo` is specified as an `AdditionalLabel`:

https://github.com/kubernetes/test-infra/blob/715d482b458945df55597a4e9ca66fafadbe42d1/prow/plugins/plugins.go#L340

Reply:
Aha! 👍